### PR TITLE
[cherry-pick] katello-tracer component implementation and Tracer fixes

### PR DESCRIPTION
**Pull request by mirekdlugosz**
Cherry-picking #748 and #758 into `stable-satellite`. Both of them, since #758 contains fixes from review for #748, which was merged early.

There was a conflict in entities during cherry-picking. Might impact other pending cherry-picks if this is merged first, or this will need rebase if something else is merged first. Shouldn't stop reviewers from ACKing and merging, we'll sort it out along the way.

### CODE REVIEW

File: nailgun/entities.py [hunk 1]
```diff
@@ -4249,17 +4319,18 @@ def path(self, which=None):
...
-            return '{0}/{1}'.format(
-                super(Host, self).path(which='base'),
-                which
-            )
-        elif which in ('bulk/add_subscriptions',):
+        elif which in ('bulk/install_content',
+                       'bulk/add_subscriptions',
+                       'bulk/remove_subscriptions',
+                       'bulk/traces',
+                       'bulk/resolve_traces'):
```
Comment (mirekdlugosz) [inline nailgun/entities.py:4333]: This removal is effect of conflict, let me know if I messed it up.

Comment (ogajduse) [reply nailgun/entities.py:4333]: Method `bulk_remove_subscriptions` is missing in the cherry-pick. Look at the path `'bulk/remove_subscriptions'`.

https://github.com/SatelliteQE/nailgun/blob/49f3c4fa27b1cc3f1f4081e152d37cee722e2488/nailgun/entities.py#L4206-L4221

**Review by ogajduse — CHANGES_REQUESTED** [review]

File: nailgun/entities.py (hunk 1 above)
Comment (ogajduse) [reply nailgun/entities.py:4333]: As the PR is related to Tracer, it would be probably better to introduce the missing method in other PR.

Comment (mirekdlugosz) [reply nailgun/entities.py]: Thanks for pointing that out. I see this method was added in #474, which has not been cherry-picked yet. I removed reference to that endpoint in this PR.

**Review by ogajduse — APPROVED** [review]

**Review by JacobCallahan — APPROVED** [review]
ACK

**Pull request merged**